src/web/src/components/Posts/Users.tsx

```diff
@@ -57,14 +57,13 @@ const Users = ({ usernames, avatarSize = 25 }: Props) => {
         {usernames?.map((user) => (
           <li key={user} className={classes.user}>
             <div>
-              <TelescopeAvatar
-                img={getUserImage(user, avatarSize)}
-                size={`${avatarSize}px`}
-                name={user}
-                action={() => {
-                  window.open(`https://github.com/${user}`, '_blank');
-                }}
-              />
+              <a href={`https://github.com/${user}`}>
```

Comment:
Let's make our avatar's a bit more consistent.  Both support wrapping in an `<a>`, but the prop is named differently: `TelescopeAvatar` wants it as a `blog` and `PostAvatar` wants it as `url`.  Can we standardize on `url` or `href` or something in both cases?

And here, let's not wrap the `TelescopeAvatar` in an `<a>`, let's do it in the component.